includes/class-sensei-usage-tracking.php

```diff
@@ -25,6 +36,18 @@ class Sensei_Usage_Tracking {
 
 	private static $job_name = 'sensei_core_jobs_usage_tracking_send_data';
 
+	/**
+	 * Initialize the class and set its properties.
+	 *
+	 * @since 1.9.20
+	 * @param string $callback_class  Callback class name
+	 * @param string $callback_method Callback method name
+	 **/
+	function __construct( $callback_class, $callback_method ) {
```

Comment:
Here, instead of callback class and method maybe better to expect any callable http://php.net/manual/en/language.types.callable.php